format/mp4/boxes.go

```diff
@@ -1367,8 +1367,45 @@ func decodeBox(ctx *decodeContext, d *decode.D, typ string) {
 		d.FieldU32("h_spacing")
 		d.FieldU32("v_spacing")
 	case "uuid":
-		d.FieldRawLen("uuid", 16*8, scalar.RawUUID, uuidNames)
-		d.FieldRawLen("data", d.BitsLeft())
+		uuidBR := d.FieldRawLen("uuid", 16*8, scalar.RawUUID, uuidNames)
```

Comment:
I wonder if a user wants sym to always be the dashed uuid string and the names mapping to set description? atm sym is only used and will be a dashed uuid string as fallback, maybe ok also? 🤔

Reply:
I kept it the same way as it used to be. I think it's fine.